frontend/survey-project/src/app/components/pages/survey/pages/survey-details/survey-details.component.ts

```diff
@@ -1,10 +1,120 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { ChartData, ChartOptions, ChartType } from 'chart.js';
+import { SurveyService } from '../../../../../shared/services/survey.service';
+import { ActivatedRoute } from '@angular/router';
+import { ToastService } from '../../../../../shared/services/toast.service';
+import { LoaderService } from '../../../../../shared/services/loader.service';
+import { SurveyResultsRequest } from '../../../../../shared/dto/requests/survey/survey-results-request';
+import { SurveyResultsResponse } from '../../../../../shared/dto/responses/survey/survey-results-response';
 
 @Component({
   selector: 'app-survey-details',
   templateUrl: './survey-details.component.html',
   styleUrl: './survey-details.component.scss'
 })
-export class SurveyDetailsComponent {
+export class SurveyDetailsComponent implements OnInit {
+  public chartType: ChartType = 'bar';
+  public chartLegend = false;
 
+  constructor(
+    private readonly surveyService: SurveyService,
+    private readonly route: ActivatedRoute,
+    private readonly toastService: ToastService,
+    private readonly loaderService: LoaderService
+  ) {}
+
+  public chartOptions: ChartOptions = {
```

Comment:
Move these configurations in const folder of sutvey-details, so that we don't have all of this in the main component (for readability).